lib/games/tables_game.dart

```diff
@@ -21,8 +22,8 @@ class _ChoiceDetail {
 }
 
 class TablesGame extends StatefulWidget {
-  final String question;
-  final String answers;
+  final BuiltList<int> question;
```

Comment:
For simplicity - since we are only using 2 operands - you can make it question1 and question2 or x and y